iotile_analytics_core/iotile_analytics/core/group.py

```diff
@@ -186,7 +186,7 @@ def find_stream(self, partial, include_empty=False):
 
         return found[0]
 
-    def fetch_stream(self, slug_or_name):
+    def fetch_stream(self, slug_or_name, allow_empty=False):
```

Comment:
In the case of `allow_empty=True`, does `fetch_stream` what is the return type of the empty stream?

Reply:
It should be an empty `StreamSeries`, test coverage is here:
https://github.com/iotile/iotile_analytics/pull/79/files#diff-be9d8aa29725634a5bce735649896aa0R99